test/cypress/support/commands.js

```diff
@@ -9,9 +9,12 @@ Cypress.Commands.add('getByDT', (element) => {
   return cy.get(`[data-test="${element}"]`)
 })
 
+const closeModalDelay = 300
+
 // NOTE: We can go a step further and not use UI to do repetitive tasks.
 // https://docs.cypress.io/guides/getting-started/testing-your-app.html#Fully-test-the-login-flow-%E2%80%93-but-only-once
 Cypress.Commands.add('giSignUp', (userName, password = '123456789') => {
+  cy.wait(closeModalDelay) // TODO: Find a way to get event from sbp
```

Comment:
@sandrina-p correct me if I'm wrong but I believe Cypress has a way of waiting for elements to appear?

Also, see comment about magic numbers: https://github.com/okTurtles/group-income-simple/pull/660/files#r324355432

Reply:
All fixed now